src/js/components/tooltip.js

```
@@ -1,119 +1,183 @@
     }
 
-};
+    const props = includes(properties[0], dir) ? properties[1] : properties[0];
+    align = props.find((prop) => elOffset[prop] === targetOffset[prop]) || 'center';
+
+    return [dir, align];
+}
+
+function parseProps(options) {
+    const { el, id, data } = options;
+    return ['delay', 'title'].reduce((obj, key) => ({ [key]: getData(el, key), ...obj }), {
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)